kuma/users/tests/test_templates.py

```diff
@@ -539,10 +572,25 @@ def test_banned_user_one_click_page_template_ban_button(self):
         ban_other_reasons = page.find('#ban-for-other-reasons')
 
         eq_(len(revisions_found), num_revisions)
-        eq_(ban_button.text(), "Revert Revisions")
+        eq_(ban_button.text(), "Submit Spam")
         eq_(len(ban_other_reasons), 0)
 
-        # For self.testuser2 (banned, has no revisions needing to be reverted)
+    def test_banned_user_no_revisions_or_already_spam_ban_button(self):
+        """
+        Test for a banned user with no revisions that can be marked as spam.
+
+        We test the ban button text for a banned user who has either
+        no revisions or revisions already marked as spam.
+        """
+        # Ban both self.testuser and self.testuser2
+        UserBan.objects.create(user=self.testuser, by=self.admin,
+                               reason='Banned by unit test.',
+                               is_active=True)
+        UserBan.objects.create(user=self.testuser2, by=self.admin,
```

Comment:
``self.testuser`` appears to be absent from the rest of the test, and the ``UserBan`` could be dropped.